src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -334,6 +334,31 @@ public static void ACos_Advanced(double real, double imaginary, double expectedR
             VerifyRealImaginaryProperties(result, expectedReal, expectedImaginary);
         }
 
+        public static IEnumerable<object[]> ACos_Legacy_TestData ()
+        {
+            // Boundary values
+            yield return new object[] { double.MaxValue, 0, double.NaN, double.NaN };
+            yield return new object[] { double.MinValue, 0, double.NaN, double.NaN };
+
+            // Invalid values
+            foreach (double invalidReal in s_invalidDoubleValues) {
```

Comment:
The documentation covering the options is also incomplete (we only cover a small subset of them currently here: https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference).

However, the following Roslyn PRs do give a decent summary of (what I believe to be) all the currently available options:
* https://github.com/dotnet/roslyn/pull/15020
* https://github.com/dotnet/roslyn/pull/15029
* https://github.com/dotnet/roslyn/pull/15065